Src/Support/Google.Apis.Auth.Tests/OAuth2/GoogleCredentialTests.cs

```diff
@@ -713,6 +713,92 @@ public async Task SignBlobAsync_UnsupportedCredential()
             await Assert.ThrowsAsync<InvalidOperationException>(() => googleCredential.SignBlobAsync(Encoding.ASCII.GetBytes("toSign")));
         }
 
+        [Fact]
+        public async Task UniverseDomain_RequestAndCredentialSame()
+        {
+            string universeDomain = "custom.domain";
+
+            HttpRequestMessage request = new HttpRequestMessage();
+            request.SetOption(ConfigurableMessageHandler.UniverseDomainKey, universeDomain);
+
+            GoogleCredential credential = GoogleCredential.FromAccessToken("fake_token")
+                .CreateWithUniverseDomain(universeDomain);
+
+            await (credential as IHttpExecuteInterceptor).InterceptAsync(request, default);
+
+            Assert.Equal("fake_token", request.Headers.Authorization.Parameter);
+        }
+
+        [Fact]
+        public async Task UniverseDomain_NoneInRequestDefaultInCredential()
+        {
+            HttpRequestMessage request = new HttpRequestMessage();
+
+            GoogleCredential credential = GoogleCredential.FromAccessToken("fake_token");
```

Comment:
Maybe just declare this using type IHttpExecuteInterceptor here, to avoid the "as" below?

Reply:
(Ditto for other tests)